src/components/updatemenus/attributes.js

```diff
@@ -49,6 +49,7 @@ var buttonsAttrs = {
 
 module.exports = {
     _isLinkedToArray: 'updatemenu',
+    _arrayAttrRegexps: [/^updatemenus\[(0|[1-9][0-9]+)\]\.buttons/],
```

Comment:
Just wondering why you chose to attach `_arrayAttrRegexps` to the attribute object? 

Alternatively, we could set in the module object [here](https://github.com/plotly/plotly.js/blob/027d97fe3f9c2fb2d5e816d41719df49eb49c763/src/components/annotations/index.js#L15) - and not having to skip it in the plot schema [logic](https://github.com/plotly/plotly.js/pull/1403/files#diff-9d405afa212832aed790fcec3d0e16f9R31).